CMakeLists.txt

```diff
@@ -59,6 +59,11 @@ if(NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE "Debug")
 endif(NOT CMAKE_BUILD_TYPE)
 
+if(NOT ${CMAKE_BUILD_TYPE} EQUAL "Debug")
```

Comment:
It seems that `STREQUAL` must be used instead here.

Reply:
Good catch 👍